src/js/intlTelInput.js.ejs

```diff
@@ -761,6 +765,33 @@ Plugin.prototype = {
     // dropdownHeight - 1 for border
     var cssTop = (!dropdownFitsBelow && dropdownFitsAbove) ? "-" + (this.dropdownHeight - 1) + "px" : "";
     this.countryList.css("top", cssTop);
+    
+    // if container, append countryList to this.$drop and this.$drop to container
+    if (this.options.container) {        
+      this.$drop = $("<div>", {
+        "class": "intl-tel-input iti-container"
+      });
+      
+      var $dropInner = $("<div>", {
+        "class": "flag-dropdown"
+      }).appendTo(this.$drop),
+        inputHeight = !dropdownFitsBelow && dropdownFitsAbove ? 0 : that.telInput.outerHeight();
+      
+      // calculate placement
+      that.$drop.css({
+        "top": pos.top + inputHeight - parseInt(this.telInput.css('borderTopWidth'), 10) - parseInt(this.telInput.css('borderBottomWidth'), 10),
+        "left": pos.left,
+        "position": "absolute"
+      });
+      
+      // close menu on resize/scroll
+      this.telInput.parents().off("resize" + this.ns + " scroll" + this.ns).on("resize" + this.ns + " scroll" + this.ns, function () {
+        if (!that.countryList.hasClass("hide")) that._closeDropdown();
+      });
+      
+      this.$drop.appendTo(this.options.container);
```

Comment:
I'm injecting the menu on every open and using `.detach()` on every close (same as remove except all jQuery data is kept associated with the menu when it is reinserted). This prevents the DOM from being polluted and works better on single page apps. Regenerating it on every open is only about a 1-2 ms difference, but I should move some of that code to _generateMarkup to completely remove that lag.

Also, since we're already using the `iti-container` class, it would make more sense to just set `position: absolute` in the CSS. I'll update my code.